src/NHSD.BuyingCatalogue.Solutions.Persistence/Repositories/SolutionFrameworkRepository.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NHSD.BuyingCatalogue.Data.Infrastructure;
+using NHSD.BuyingCatalogue.Solutions.Contracts.Persistence;
+using NHSD.BuyingCatalogue.Solutions.Persistence.Models;
+
+namespace NHSD.BuyingCatalogue.Solutions.Persistence.Repositories
+{
+    public sealed class SolutionFrameworkRepository : ISolutionFrameworkRepository
+    {
+        private const string GetFrameworkBySolutionIdSql = @"
+            SELECT f.Id,
+                   f.ShortName as FrameworkName
+            FROM dbo.FrameworkSolutions AS fs
+                 INNER JOIN dbo.Framework AS f 
+                    ON fs.FrameworkId=f.Id
+            WHERE fs.SolutionId = @solutionId;";
```

Comment:
You'll probably want to remove or update the alias for `ShortName` here based on my comment above about `SolutionFrameworkListResult`.

There should be spaces around operators so on line 18 you just need to add spaces around `=`.

Reply:
Removed alias, as property is updated to `ShortName`